.travis.yml

```diff
@@ -1,3 +1,9 @@
 language: ruby
 rvm:
   - 2.1.0
+
+branches:
```

Comment:
No we don't need to because all other branches that are getting outdated the test suite works for sure because we are maintaining running tests since stoney.